index.html

```diff
@@ -98,19 +395,46 @@ <h2>Year: <label id='active-year'>1997</label></h2>
 			  'circle-opacity': 0.8
 			}
 		  });
-		  
+
+				//add data to power the zoom control
+		  		addVectorLayer(
+					map,
+					{
+						'sourceName': 'texas-school-districts', // data source name for internal use
+						'sourceID': 'texas_school_districts_v2', // name of the Mapbox layer from which the data will be loaded
+						'sourceURL': 'mapbox://core-gis.e4af0de1', // Mapbox URL
+						'lineLayerName': 'texas-school-districts-lines', // OPTIONAL name we'll use for the layer that shows the outlines. Leave out or set to false if you don't want outlines displayed.
+						'lineColor': '#cdcecb', // colour to draw those outlines with; safe to leave out if we're not drawing outlines, but must be explicitly set if we are
+						'legendID': 'texas_school_districts', // OPTIONAL: the id in the legend, so we can set it to active or inactive as appropriate. Simply leave out for layers that don't appear in the legend
+						'displayBehind': 'waterway-label', // ID of another existing layer, which Mapbox will make sure this one gets drawn behind
+						'polygonLayerName': 'texas-school-districts-poly', // OPTIONAL name we'll use for the layer that invisibly stores the polygon extents. Needed if we're either going to add this layer to either the zoom to districts control or set click events (e.g. popups) on it.	Leave out or set to false if you don't want one.
+						'polygonFillColor': 'rgba(200, 100, 240, 0)', // colour to fill polygons with. Needed if there's going to be a polygon layer; simply leave out if not.
+						'polygonOutlineColor': 'rgba(200, 100, 240, 0)', // colour to draw polygon boundaries with. Needed if there's going to be a polygon layer; simply leave out if not.
+						'visibleOnLoad': true, // set this optional argument to true to have the layer visible on load. Leave out or set to false to have it hidden on load
+						'usedInZoomControl': true // set this optional argument to true if this layer will be used in the Zoom to Districts control, otherwise leave it out or set it to false.
+					}
+				);
+
 		  //add interactivity to the time slider
 		  document.getElementById('slider').addEventListener('input', function(e) {
 			  var year = parseInt(e.target.value);
 			  // update the map
 			  map.setFilter('campuses', ['==', ['number', ['get', 'year']], year]);
-			  
+
 			  // update text in the UI
 			  document.getElementById('active-year').innerText = year;
 
 			});
-		});
-		
+
+			runWhenLoadComplete();
```

Comment:
This is the only thing you had forgotten to do: having got all the other code in place, you just weren't actually calling it on map load.

Reply:
Ha! Awesome. I knew it was something simple, thank you for adding that.